.travis.yml

```diff
@@ -2,7 +2,7 @@ dist: trusty
 sudo: required
 language: java
```

Comment:
I'm not sure that all these are correct. The correct versions of these properteis shoud be

dist: xenial
language: java
os: linux
jdk:
  - openjdk17